source/default/_patterns/01-atoms/svg/sprite-svg.css

```diff
@@ -0,0 +1,3 @@
+[class^='sprite-'] {
+  @apply w-6 h-6 inline-block;
```

Comment:
This does not allow for changing the size of an icon when used in dom. Like `<i class="sprite-close w-2"></i>` will not affect the width. The ^^ css has a higher specificity than `w-2`.

Reply:
Noted. Looking at a solution here now.

Reply:
Check this file and the demo now. Should be resolved.